specification/columns/commitmentdiscountusage.md

```diff
@@ -0,0 +1,38 @@
+# Commitment Discount Usage
+
+Commitment Discount Usage indicates whether the charge corresponds with the consumption of the [*commitment-based discount*](#glossary:commitment-based-discount) identified in the CommitmentDiscountId column or the unused portion of the committed amount.
+
+The CommitmentDiscountUsage column MUST be present in the billing data. This column MUST be of type String, MUST be null when [CommitmentDiscountId](#commitmentdiscountid) is null, and MUST NOT be null when CommitmentDiscountId is not null. The CommitmentDiscountCategory MUST be one of the allowed values.
```

Comment:
```suggestion
The CommitmentDiscountUsage column MUST be present in the billing data when the provider supports *commitment-based discounts*. This column MUST be of type String, MUST be null when [CommitmentDiscountId](#commitmentdiscountid) is null, and MUST NOT be null when CommitmentDiscountId is not null. The CommitmentDiscountCategory MUST be one of the allowed values.
```
Reminder: 
Noticed that a conversation regarding the Feature level constraint is marked as resolved, but the 'Column required' still present.